lib/vtls/bearssl.c

```diff
@@ -340,8 +365,29 @@ static CURLcode bearssl_connect_step1(struct Curl_easy *data,
     return CURLE_SSL_CONNECT_ERROR;
   }
 
+  if(ca_info_blob) {
+    struct cafile_source source = {
+      .type = CAFILE_SOURCE_BLOB,
+      .data = ca_info_blob->data,
+      .len = ca_info_blob->len,
```

Comment:
I might be undecaffeinated, but isn't this C99 syntax?

Reply:
Right. Changed to the older way